src/services/EnvironmentVariables.ts

```diff
@@ -213,8 +215,10 @@ export class EnvironmentVariables {
 					!this.ISSUER || this.ISSUER.trim().length === 0 ||
 					!this.TXMA_QUEUE_URL || this.TXMA_QUEUE_URL.trim().length === 0 ||
 					!this.YOTI_KEY_SSM_PATH || this.YOTI_KEY_SSM_PATH.trim().length === 0 ||
-					!this.YOTIBASEURL || this.YOTIBASEURL.trim().length === 0) {
-					logger.error("Environment variable PERSON_IDENTITY_TABLE_NAME or YOTI_SDK or YOTICALLBACKURL or ISSUER is not configured");
+					!this.YOTIBASEURL || this.YOTIBASEURL.trim().length === 0 ||
+					!this.KMS_KEY_ARN || this.KMS_KEY_ARN.trim().length === 0 ||
+					!this.DNSSUFFIX || this.DNSSUFFIX.trim().length === 0) {
+					logger.error("Environment variable PERSON_IDENTITY_TABLE_NAME or YOTI_SDK or KMS_KEY_ARN or ISSUER is not configured");
```

Comment:
Shall we add `DNSSUFFIX` to this error log?

Reply:
Done